srcdoc/install.mt

```diff
@@ -72,6 +72,12 @@ It is turned on by default if the prerequisites (<a href="https://www.gnu.org/so
 This option can be used to add or override the compile options being passed to the C compiler.
 As an example, <a href="https://en.wikipedia.org/wiki/AddressSanitizer">AddressSanitizer (ASan)</a> can be enabled when using recent versions of GCC or Clang, by passing <code>-DCMAKE_C_FLAGS="-fsanitize=address -fno-stack-protector -fno-omit-frame-pointer"</code>.
 </dd>
+<dt><code>-DCMAKE_BUILD_TYPE=Release|Debug</code></dt>
+<dd>
+This option specifies the build type, <code>Release</code> or <code>Debug</code>.
+The default is <code>Release</code>.
+You'd like to add some more debugging flags to `CMAKE_C_FLAGS` on <code>Debug</code> build, for example <code>-DCMAKE_BUILD_TYPE=Debug -DCMAKE_C_FLAGS=-g3</code> if you use <code>gdb</code>.
```

Comment:
The suggestion about using `-g3` is interesting.

I actually wonder if we should simply set `-g3` as the default of CMAKE_C_FLAGS (we use `-g` now). WDYT?

Reply:
It would be useful but I'm worried about artifact sizes --  `-g3` could make the h2o executable file much larger.

So I tested how much the `h2o` binary size increases:

* `-DWITH_MRUBY=off` (i.e. with `-g`): 11M
* `-DCMAKE_C_FLAGS=-g3 -DWITH_MRUBY=off`: 12M

The difference is so small that setting `-g3` as the default has great benefit and little disadvantage.

Can I change the default option in this PR? Or should I make a separate PR for it?

Reply:
Thank you for checking the difference. Now I am confident that the default can be changed to `-g3`.

I'd appreciate a separate PR.

Reply:
Thanks! 

Just removed the example of -DCMAKE_C_FLAGS=-g from the doc: https://github.com/h2o/h2o/pull/2261/commits/b4b0d8dd57a9c050d8bb616cab34739a806048dc

Will make a request for `-g3`.